.github/workflows/ci.yml

```
@@ -0,0 +1,231 @@
+        uv run task format --check
+        uv run task demo
+        
+    - name: Integration test suite
+      run: uv run python scripts/run_tests.py integration
+
+    - name: Coverage report
+      run: uv run python scripts/run_tests.py coverage
+
+    - name: Upload coverage reports
+      uses: codecov/codecov-action@v4
+      if: matrix.python-version == '3.13'
```

actionlint (1.7.7)

property "python-version" is not defined in object type {}

(expression)